<launch>

    <arg name="use_gripper" default="false"/>

    <node pkg="intel_aero_ros" type="gripper_test_node" name="gripper_test_node" output="screen">
        <remap from="lengths" to="/gripper/lengths"/>
        <param name="total_time" value="5.0"/>
        <rosparam param="start_position">[0.0, -1.0, 1.5]</rosparam>
        <rosparam param="offset">[0, 0, -0.160]</rosparam>
        <rosparam param="end_velocity">[0, 0.5, -0.1]</rosparam>
    </node>

    <node if="$(arg use_gripper)" pkg="intel_aero_ros" type="gripper_node" name="gripper" output="screen">
        <remap from="lengths" to="/gripper/lengths"/>
    </node>

</launch>
